frontend/src/locales/en-CA.json

```diff
@@ -33,6 +33,16 @@
     "supervisor": "SUPERVISOR",
     "admin": "ADMIN"
   },
+  "projectView": {
+    "createdBy": "Created by",
+    "enquiryID": "Enquiry ID",
+    "inapplicableSubmissionType": "A Navigator has reviewed your submission and found it inapplicable to Housing. For questions, please submit a general enquiry.",
+    "listEmpty": "Your project related enquiries will be displayed here",
+    "of": "of",
```

Comment:
Semantics: Looks like this is being used in the context of "Page 1 of 3". However, I don't know if all languages really have the concept of an "of". It might be better to rename this as `rangeSeparator` or something similar so that a developer knows what the context of use is, without having it be bound to a specific language construct.

Reply:
done